src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -885,11 +881,23 @@ extern "C" UInt32_BOOL ResetEvent(HANDLE event)
     return UInt32_TRUE;
 }
 
-extern "C" UInt32 GetEnvironmentVariableA(const char * pName, char * pBuffer, UInt32 size)
+extern "C" UInt32 GetEnvironmentVariableA(const char * name, char * buffer, UInt32 size)
 {
-    // UNIXTODO: Implement this function
-    *pBuffer = '\0';
-    return 0;
+    const char* value = std::getenv(name);
```

Comment:
I guess the `std::` is needed because of we only include `cstdlib` globally. I am wondering whether we should  include the C `stdlib.h` here so that we can call this without the `std::` prefix? 

I am looking for consistency. It would be unfortunate to in a place where half of the C runtime methods are called with `std::` prefix and other half without it.